pkg/tracing/otlp/otlp.go

```diff
@@ -25,6 +26,9 @@ import (
 const (
 	TracingClientGRPC string = "grpc"
 	TracingClientHTTP string = "http"
+	AlwaysSample      string = "alwayssample"
+	NeverSample       string = "neversample"
+	RatioBasedSample  string = "traceidratiobased"
```

Comment:
Two suggestions:
1) Make this `camelCase` to make perhaps make it a tad more natural for Go / readable
2) I think we still miss support for couple more built in samplers - we should have these 3, **plus** each of these with parent based sampler. A good list for comparison can be taken from this Python SDK documentation: https://opentelemetry-python.readthedocs.io/en/latest/sdk/trace.sampling.html#module-opentelemetry.sdk.trace.sampling